src/site/markdown/httpcomponents-client-5.5.x/migration-guide/migration-to-classic.md

```diff
@@ -149,3 +149,27 @@ The 5.x `URIBuilder` in [httpcomponents-core](https://hc.apache.org/httpcomponen
 public `normalizeSyntax` method, but it strives for more thorough support of RFC 3986, 
 specifically percent-encoding all components. 
 Since 5.3, `normalizeSyntax` has been deprecated and renamed to `optimize` to emphasize the difference in behaviour.
+
+## Migration recipes
```

Comment:
@winfriedgerlach Could you please add a note that the list of recipes is by no means complete and that the users are encouraged to contribute other recipes based on their experience?